src/f32x4_.rs

```diff
@@ -518,33 +518,23 @@ impl f32x4 {
       }
     }
   }
-  // On i586, `f32::trunc` in the standard library will give wrong results.
-  #[cfg(any(
-    all(
-      any(target_arch = "x86", target_arch = "x86_64"),
-      target_feature = "sse"
-    ),
-    all(
-      not(any(target_arch = "x86", target_arch = "x86_64")),
-      feature = "std"
-    ),
-  ))]
+  #[cfg(feature="std")]
```

Comment:
This needs to allow for no_std with sse enabled:
`#[cfg(any(target_feature="sse", feature="std"))]`

Reply:
And the similar configs